### ISSUE TITLE
site not displaying correctly when run on nginx

### ISSUE THREAD

**Issue by GerBreOwn** [darklow/django-suit#691]
Greetings:
I have just started using Django Suit.  It looks great when I run the Django server.  However when I run it with the nginx server all formating is lost.  It looks like it is running under gunicorn.
Any ideas on how I can fix this?
Thanks,
Gerald S. Brown

**Comment by darklow** [comment]
In production mode, you have to serve static files separately:
https://docs.djangoproject.com/en/2.0/howto/static-files/

**Comment by GerBreOwn** [comment]
I followed the link you provide but I am still getting the same results.

I "/thought"/ Nginx serves ONLY Django static file and Gunicorn serves 
Django dynamic files.

I went to the Nginx website but didn't see anything about static files.

Prior to installing Django-Suit I did not have this problem. It was 
running on Nginx then and everything looked fine.

I really like the look of Django-Suit so I need to get it running 
correctly so any additional pointers will be greatly appreciated.

Thanks.
